test/03_USM.test.js

```diff
@@ -29,127 +43,280 @@ contract('USM', (accounts) => {
+        // - Since USM.BUY_SELL_ADJUSTMENTS_HALF_LIFE = 60 (1 min), 3 min should mean a decay of ~1/8.
+        // - The way our "time decay towards 1" approximation works, decaying 4 by ~1/8 should yield 1 + (4 * 1/8) - 1/8 = 1.375.
+        //   (See comment in USM.mintBurnAdjustment().)
+        // - And finally - since our decay's exponentiation is approximate (WadMath.wadHalfExp()), don't compare fundDefundAdj4 to
+        //   targetFundDefundAdj4 *exactly*, but instead first round both to the nearest multiple of 10 (via divRound(10)).
+        const block0 = (await web3.eth.getBlockNumber())
+        const t0 = (await web3.eth.getBlock(block0)).timestamp
+        const timeDelay = 3 * 60
+        await timeMachine.advanceTimeAndBlock(timeDelay)
+        const block1 = (await web3.eth.getBlockNumber())
+        const t1 = (await web3.eth.getBlock(block1)).timestamp
+        t1.toString().should.equal((t0 + timeDelay).toString())
+        const fundDefundAdj4 = (await usm.fundDefundAdjustment())
+        const decayFactor4 = WAD.div(EIGHT)
+        const targetFundDefundAdj4 = WAD.add(wadMul(fundDefundAdj3, decayFactor4)).sub(decayFactor4)
+        fundDefundAdj4.divRound(TEN).toString().should.equal(targetFundDefundAdj4.divRound(TEN).toString())
       })
 
-      describe('with existing FUM supply', () => {
+      describe("with existing FUM supply", () => {
         beforeEach(async () => {
           await usm.fund(user1, user1, oneEth, { from: user1 })
         })
 
-        it('allows minting USM', async () => {
-          const fumBuyPrice = (await usm.fumPrice(sides.BUY))
-          const fumSellPrice = (await usm.fumPrice(sides.SELL))
-
+        it("allows minting USM", async () => {
           await usm.mint(user1, user2, oneEth, { from: user1 })
-          const usmBalance = (await usm.balanceOf(user2))
-          usmBalance.toString().should.equal(oneEth.mul(priceWAD).div(WAD).toString())
-
-          const newFumBuyPrice = (await usm.fumPrice(sides.BUY))
-          newFumBuyPrice.toString().should.equal(fumBuyPrice.toString()) // Minting doesn't change the fum price if buffer is 0
-          const newFumSellPrice = (await usm.fumPrice(sides.SELL))
-          newFumSellPrice.toString().should.equal(fumSellPrice.toString())
+          const ethPool2 = (await weth.balanceOf(usm.address))
+          ethPool2.toString().should.equal(oneEth.mul(TWO).toString())
+
+          const usmBalance2 = (await usm.balanceOf(user2))
+          // Verify that sliding prices are working: this mirrors the sliding FUM prices above - see the math in USM.usmFromMint(),
+          // with mint() here 2xing the pool (1 ETH from fund() earlier -> 2) and the initial USM price of $1 = 0.004 ETH, the USM
+          // added should be 1 (current ETH pool) / 0.004 * (1 - 1 / 2) = 125 (ie, the ETH price $250 / 2):
+          usmBalance2.toString().should.equal(wadMul(oneEth, priceWAD).div(TWO).toString())
```

Comment:
Can we code the target value into a formula?

Reply:
Done.